vizro-ai/docs/pages/tutorials/quickstart-dashboard.md

```diff
@@ -88,12 +88,37 @@ Next, submit the data and prompt string:
 dashboard = vizro_ai._dashboard([df], user_question)
 ```
 
-The call to `_dashboard()` triggers the dashboard building process. Once Vizro-AI finishes the dashboard generation process, you can launch the dashboard with `build()`.
+The call to `_dashboard()` triggers the dashboard building process.
+
+## 7. Build dashboard
+Once Vizro-AI finishes the dashboard generation process, launch the dashboard with `build()`.
+
```

Comment:
This section is confusing me -- sorry. 

In line 91, we have a call to `_dashboard()` which triggers the "building process".

Then line 94 says "Once Vizro-AI finishes the dashboard generation process" and that you need to call `build()`. 

But didn't line 91 kick off the build according to the text. What's the difference between generation and build?

The names seem odd if the one that doesn't include the word "build" does the building and the one that does have "build" is the launcher. 

Could you clarify for me please @nadijagraca ?

Reply:
Thanks for pointing out, very good point! So to be more specific, 

`_dashboard()` generates the Vizro Dashboard object.

`Vizro().build(dashboard)` builds the dashboard.

Reply:
As Lingyi mentioned below, `_dashboard()` generates the Vizro Dashboard object, but we need to do `Vizro().build(dashboard).run()` to run the dashboard.